src/app/navbar/navbar.component.ts

```diff
@@ -12,9 +12,18 @@ export class NavbarComponent implements OnInit {
     @Output() changeView: EventEmitter<string> = new EventEmitter();
 
     constructor(private _eventService: EventService, private _categService: CategoryService) { }
-    ngOnInit() { }
-
+    temperature: string;
```

Comment:
Make these explicitly public variables under the export statement

e.g.
export class NavbarComponent implements OnInit {
    public icon: string;